test/run

```diff
@@ -431,6 +434,15 @@ kill_test_application
 echo "Testing incremental build"
 test_incremental_build
 
+# test express webapp
+prepare express-webapp
+ct_s2i_build_as_df file://${test_dir}/test-express-webapp ${IMAGE_NAME} ${IMAGE_NAME}-testexpress-webapp ${s2i_args} $(ct_build_s2i_npm_variables)
```

Comment:
Sorry, I am a nit-picker. Would it be possible to have it in alone function? Like here: https://github.com/sclorg/s2i-nodejs-container/blob/206dddf65e377bdc2008114c36c78d4ee51c4265/test/run#L63

Reply:
Sure, will do.

Reply:
Sure, will do.

Reply:
Done now.